### PR TITLE
Added ability to specify multiple comma separated permissions files.

### PR DESCRIPTION
**Pull request by ankanm**
* Added support for multiple permissions file.

@markg-moz

### CODE REVIEW

File: aaad/authorizers.py [hunk 1]
```diff
@@ -175,9 +175,11 @@ def filter_response(self, resource, data, actas):
             return framework.filterResponseBody(data, allowed_namespaces, resource)
 
         def _parse_permissions_file(self, filename):
-            with open(filename, 'r') as data_file:
-                return yaml.load(data_file)
-            return ''
+            permissions = {}
+            for item in filename.split(','):
```
Comment (markg-moz) [inline aaad/authorizers.py]: filename should probably be plural now, or something generic like file_spec.

Comment (markg-moz) [inline aaad/authorizers.py]: A trailing comma would mess it up (e.g. `file1.yml,file2.yml,`)  so an `if item.tstrip():` might be useful.  Although if it fails loudly it's probabyl OK.

File: aaad/utils.py [hunk 2]
```diff
@@ -44,3 +44,12 @@ def getAllowedNamespacePatterns(act_as, permissions):
 
     return allowed_namespace_patterns
 
+def merge_dicts(dict1, dict2):
```
Comment (markg-moz) [inline aaad/utils.py]: There's a `.update()` method on all dictionaries, so you can do `existing_dict.update(dict_with_more_stuff)`.

Comment (vmahedia) [inline aaad/utils.py]: @markg-moz that would be wrong here, as it will override the value and not merge.

Comment (ankanm) [inline aaad/utils.py]: Right. So, I'm keeping this as is.

File: aaad/authorizers.py (hunk 1 above)
Comment (ankanm) [inline aaad/authorizers.py]: Done.

File: aaad/utils.py [hunk 3]
```diff
@@ -44,3 +44,12 @@ def getAllowedNamespacePatterns(act_as, permissions):
...
+    ''' merges dict2 into dict1 and returns dict1 '''
+    if isinstance(dict1, dict) and isinstance(dict2, dict):
+        for k, v in dict2.iteritems():
+            if k not in dict1:
+                dict1[k] = v
+            else:
+                dict1[k] = merge(dict1[k], v)
```
Comment (markg-moz) [inline aaad/utils.py]: Where is the merge function defined and what does it do?

File: aaad/authorizers.py [hunk 4]
```diff
@@ -174,10 +174,14 @@ def filter_response(self, resource, data, actas):
...
             return framework.filterResponseBody(data, allowed_namespaces, resource)
 
-        def _parse_permissions_file(self, filename):
-            with open(filename, 'r') as data_file:
-                return yaml.load(data_file)
-            return ''
+        def _parse_permissions_files(self, filenames):
+            permissions = {}
+            for item in filenames.split(','):
+                filename = item.strip(' ,') # strip ',' and ' '
```
Comment (markg-moz) [inline aaad/authorizers.py]: You only need `.strip()` since the commas will already be gone.  Also `strip()` strips on all whitespace not just space.

Comment (ankanm) [inline aaad/authorizers.py]: Done.

**Comment by markg-moz** [comment]
LGTM

**Pull request merged**